src/store/traction/pacbio/runs/mutations.js

```diff
@@ -6,14 +8,23 @@ const mutateRun = (key) => (state, val) => {
   state.currentRun[key] = val
 }
 
+const mutateWellDefaults = (key) => (state, val) => {
+  Vue.set(state.currentRun.wellDefaults, key, val)
```

Comment:
What does `Vue.set` do here?

Reply:
> What does `Vue.set` do here?

Vue.set is a tool that adds a new property to an already reactive property and makes sure that property is also reactive. If you didn't use it the property would not show up in the ui. In Vue 3 it is no longer needed as you can make an object reactive and any new properties are reactive by default.